core/common/src/main/java/alluxio/uri/Authority.java

```diff
@@ -23,7 +23,8 @@
  */
 public interface Authority extends Comparable<Authority>, Serializable {
   Logger LOG = LoggerFactory.getLogger(Authority.class);
-  Pattern ZOOKEEPER_AUTH = Pattern.compile("^zk@(.*)");
+  Pattern SINGLE_MASTER_AUTH = Pattern.compile("^([^:,;]+):(\\d+)$");
```

Comment:
Did this fix a bug? Is a test case added for this?

Reply:
This does not fix a bug, it just makes the check be more concrete and accurate.
In the previous way, we do not check if the port is an integer.

Some unit tests have been added in the `AuthorityTest`.

```
assertTrue(Authority.fromString("localhost:") instanceof UnknownAuthority);
assertTrue(Authority.fromString("localhost:asdsad") instanceof UnknownAuthority);
```
Those will be passed in the previous way as a SingleMasterAuthority but not in the new way.